package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@segment/integration-appsflyer",
-  "version": "1.1.2",
+  "version": "1.1.3",
```

Comment:
i think this should be a minor bump rather than patch. it wasn't really a bug original, this just expands support